README.md

```diff
@@ -14,3 +14,15 @@ See https://aur.archlinux.org/packages/?K=zivid for the list of Zivid packages.
 These packages can easily be installed by using one of the [AUR helpers](https://wiki.archlinux.org/index.php/AUR_helpers) like [yaourt](https://aur.archlinux.org/packages/yaourt).
 
     yaourt -S --noconfirm zivid-studio zivid-tools
+
+## Uploading new versions to AUR
+
+1. Update `zividVersion` in `scripts/generate_all_pkgbuilds.sh`
+2. Run the tests. See [.travis.yml](.travis.yml).
+3. Run `./scripts/update_aur.sh`.
+
+### Register new packages
```

Comment:
- Should say "registering", to keep in line with the other headlines?
- Is it intentional that this is a sub-section of "Uploading new versions to AUR", or should we use `##` instead of `###`

Reply:
I was intentional, but probably not for a good reason. Ill fix both things.